Libplanet.Tests/Action/PolymorphicActionTest.cs

```diff
@@ -87,6 +87,21 @@ public void DetectLackOfActionType()
             );
         }
 
+        [Fact]
+        public void PolymorphicActionToString()
+        {
+            Assert.Equal(
+                "Libplanet.Action.PolymorphicAction`1[Libplanet.Tests.Common.Action.Sleep]",
```

Comment:
That <code>`1</code> in itself means it has one type parameter… Instead of `GetType().Name`, how about using simply `nameof(PolymorphicAction)`?

Reply:
Originally, I wanted to follow the original format used from `ToString`. Of course, I will apply your suggestion but one more thing, I wonder your opinion about using and `[]` instead `<>` like code.

Reply:
That would be much better.